libfreerdp/core/credssp_auth.c

```diff
@@ -151,6 +151,48 @@ BOOL credssp_auth_setup_auth_data(rdpCredsspAuth* auth, const SEC_WINNT_AUTH_IDE
 	return TRUE;
 }
 
+static BOOL credssp_auth_client_init_cred_attributes(rdpCredsspAuth* auth)
+{
+	SECURITY_STATUS status;
+
+	WINPR_ASSERT(auth);
+
+	if (auth->kerberosSettings.kdcUrl)
+	{
+#ifdef UNICODE
+		SecPkgCredentials_KdcUrlW secAttr = { NULL };
+		ConvertToUnicode(CP_UTF8, 0, auth->kerberosSettings.kdcUrl, -1, &secAttr.KdcUrl, 0);
```

Comment:
Missing return check

Reply:
see right below, it checks if the output pointer is NULL, and it was initialized to NULL before the call. it doesn't check the return value from ConvertToUnicode, but it doesn't matter because it checks if ConvertToUnicode set a pointer value. We also only call this code if auth->kerberosSettings.kdcUrl is non-null